src/Nethermind/Nethermind.Facade/Eth/IFromTransaction.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Nethermind.Facade.Eth;
 
-public interface IFromTransaction<out T> : ITxTyped where T : TransactionForRpc
+public interface IFromTransaction<out T> where T : TransactionForRpc
```

Comment:
why seperate?

Reply:
It was separated to allow the `TransactionForRpc.TransactionConverter` to also implement `IFromTransaction<T>` for consistency. Since that class will be hidden (`internal`) I don't think this change is needed. Reverted.